tests/e2e/test_end_to_end.py

```diff
@@ -0,0 +1,2502 @@
+import os
+
+import intake
+import numpy as np
+import pytest
+
+from access_nri_intake.cli import build
+
+from .conftest import here
+
+e2e = pytest.mark.skipif(
+    "not config.getoption('--e2e')",
+)
```

Comment:
Idle curiosity - does this cause the option to get auto-documented somewhere, or do we need to remember to do that manually?

Reply:
Turns out it does - `pytest --help` returns 
```bash
...

Custom options:
  --e2e                 Run end-to-end tests

...
```

Pleasantly surprised by that one - but I'll add it to the docs too.